js/private/js_run_binary.bzl

```diff
@@ -176,16 +186,33 @@ def js_run_binary(
     if data != None:
         fail("Use srcs instead of data in js_run_binary: https://github.com/aspect-build/rules_js/blob/main/docs/js_run_binary.md#js_run_binary-srcs.")
 
-    # Copy srcs to bin
     extra_srcs = []
+
+    # Hoist js provider files to DefaultInfo
+    make_js_filegroup_target = (include_transitive_sources or
+                                include_declarations or
+                                include_npm_linked_packages)
+    if make_js_filegroup_target:
+        js_filegroup_name = "{}_js_filegroup".format(name)
+        _js_filegroup(
+            name = js_filegroup_name,
+            srcs = srcs,
+            include_transitive_sources = include_transitive_sources,
+            include_declarations = include_declarations,
+            include_npm_linked_packages = include_npm_linked_packages,
+            tags = kwargs.get("tags"),
```

Comment:
should this use https://docs.aspect.build/aspect-build/bazel-lib/v1.5.0/docs/utils-docgen.html#propagate_well_known_tags rather than just "tags"?

Reply:
in this case, I want to propagate all tags since I'm not looking for just exectution requirements. If a user tags the js_run_binary as manual then all targets it stamps out should get that too